src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -11,7 +11,16 @@
 #include <bitmap.h>
 
 /* Assembly function for sinv from new component */
-extern word_t hypercall_entry_rets_inv(spdid_t cur, int op, word_t arg1, word_t arg2, word_t *ret2, word_t *ret3);
+extern word_t hypercall_entry_rets_inv(spdid_t cur, int op, word_t arg1, word_t arg2, word_t arg3, word_t *ret2, word_t *ret3);
+
+/*Assembly function for sinv for the fault handlers */
+extern word_t fh_div_by_zero_err_inv(unsigned long sp, unsigned long ip, unsigned long fault_addr, unsigned long fault_type);
```

Comment:
We should chat about these names.  A few high-level initial thoughts:

1. the `_inv` seems superfluous,
1. I'm not sure if `fh` provides enough of a mnemonic queue as to what these are, 
1. I think that some of these names can be made either a little more terse, or descriptive, but we can talk about it.

Naming is really hard, so it is worth us spending time on getting this right.

Also:  We likely want this to be an interface, so make it standard across components.